segmentstore/server/src/test/java/io/pravega/segmentstore/server/attributes/AttributeIndexTests.java

```diff
@@ -771,13 +837,39 @@ public void close() {
 
         private class TestStorage extends AsyncStorageWrapper {
             private final SyncStorage wrappedStorage;
+            private final Map<String, Long> startOffsets;
             private WriteInterceptor writeInterceptor;
             private SealInterceptor sealInterceptor;
             private ReadInterceptor readInterceptor;
 
             TestStorage(SyncStorage syncStorage, Executor executor) {
                 super(syncStorage, executor);
                 this.wrappedStorage = syncStorage;
+                this.startOffsets = new ConcurrentHashMap<>();
+            }
+
+            @Override
+            public CompletableFuture<Void> truncate(SegmentHandle handle, long offset, Duration timeout) {
+                // We need to simulate the ChunkStorageProvider (correct) behavior for truncating segments. While the
```

Comment:
Nit:  The new name is now - `ChunkedSegmentStorage`

Reply:
Fixed.